crates/vm/tests/integration_test.rs

```diff
@@ -133,9 +134,22 @@ fn test_vm_override_executor_height() {
     // Test getting heights.
     let vm_config = NativeConfig::aggregation(8, 3);
 
+    let vm = VirtualMachine::new(e, vm_config.clone());
```

Comment:
you copy pasted approx the same code 4 times in this file -- is there a better way?

Reply:
i'll fix this during the ctx rewrite